dags/transportation/dockless/move_table_to_schema.py

```diff
@@ -0,0 +1,51 @@
+# Basic script to move specific tables from 'public' to another schema using a binding library.
```

Comment:
Let's include this file, but then remove it later so that it's in version control but not a maintenance burden.